bundler/helpers/lib/functions/conflicting_dependency_resolver.rb

```diff
@@ -20,6 +20,7 @@ def conflicting_dependencies
       parent_specs.map do |spec|
         req = spec.dependencies.find { |bd| bd.name == dependency_name }
         {
+          "explanation" => "#{spec.name} (#{spec.version}) requires #{dependency_name} (#{req.requirement})",
```

Comment:
Should we align these to npm/yarn? Any reason they are different?
```suggestion
          "explanation" => "#{spec.name}@#{spec.version} requires #{dependency_name}@#{req.requirement}",
```

Reply:
Hm might look weird with bundler notation though, e.g. `activesupport@= 5.0.0`, thoughts about aligning npm/yarn with bundler?

Reply:
npm semver could also get real weird, e.g. `objnest@^2 <2.2 || > 2.3`

Reply:
Leaning towards unifying around the bundler format

Reply:
I tried to pick notations that are common/native to the ecosystem, although maybe the `@` notation is only common in yarn: `etag@^1.0.0`, and not so much in npm.

Reply:
Ah yeah good point, `@` is definitely more common to npm/yarn and "feels" more in line with those ecosystems. I'm up for keeping the current format as is 👍

Reply:
Another idea would be to always `@` exact versions and have requirements in brackets for all ecosystems to handle weird cases where the requirement is unruly

`dummy-pkg-b@1.0.0 requires dummy-pkg-a (< 2.0.0 || > 1.5.0) `

Reply:
Might be more clear to just use the `(1.0.0)` format everywhere in that case?

Reply:
> Yarn does use the `@` notation for that though:

Good point, forgot it's in the lockfile format 👍  yeah seems fine to keep as is tbh, nice to align format to package manager